lib/carrierwave/storage/fog.rb

```diff
@@ -0,0 +1,13 @@
+CarrierWave.configure do |config|
+  config.fog_credentials = {
+    :provider               => 'AWS',                        # required
+    :aws_access_key_id      => 'xxx',                        # required
+    :aws_secret_access_key  => 'yyy',                        # required
+    :region                 => 'eu-west-1',                  # optional, defaults to 'us-east-1'
+    :host                   => 's3.example.com',             # optional, defaults to nil
+    :endpoint               => 'https://s3.example.com:8080' # optional, defaults to nil
+  }
+  config.fog_directory  = 'name_of_directory'                          # required
+  config.fog_public     = false                                        # optional, defaults to true
+  config.fog_attributes = {'Cache-Control'=>"max-age=#{365.day.to_i}"} # optional, defaults to {}
```

Comment:
Line is too long. [97/80]<br>Surrounding space missing for operator '=>'.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.